shared/plugins_manager/src/airflow_shared/plugins_manager/plugins_manager.py

```diff
@@ -0,0 +1,280 @@
+#
```

Comment:
Maybe do it as I did with modules_loading and move all that to `__init__.py`  -  i found it much nicer as you would not have to repeat `plugins_manager` twice in imports.